core/src/main/scala/org/dbpedia/extraction/mappings/InfoboxMappingsExtractor.scala

```diff
@@ -0,0 +1,126 @@
+    val propertyParserFuncionsMappings = getTemplateMappingsFromPropertyParserFunc(propertyParserFuncions)
+
+    val invokeFunc = parserFunctions.filter(p => p.title.equalsIgnoreCase("#invoke"))
+    val wikidataParserFunc = invokeFunc.filter(p => p.children.headOption.get.toPlainText.toLowerCase.startsWith("wikidata"))
+    val propertyLinkParserFunc = invokeFunc.filter(p => p.children.headOption.get.toPlainText.toLowerCase.startsWith("propertyLink"))
+
+    val mappingQuads = propertyParserFuncionsMappings.map( p => {
+      val value = p._1.toString + "=>" + p._2.toString
+      new Quad(context.language, mapDataset, subjectUri, templateParameterProperty,
+        value, page.sourceUri, context.ontology.datatypes("xsd:string")) })
+
+    val parserFuncQuads = (propertyParserFuncions ++ wikidataParserFunc ++ propertyLinkParserFunc).map( p =>
+      new Quad(context.language, hintDataset, subjectUri, templateParameterProperty,
+        p.toWikiText, page.sourceUri, context.ontology.datatypes("xsd:string"))
+    )
+
+    val templateQuads = ExtractorUtils.collectTemplatesFromNodeTransitive(page)
+      .filter(t => List("conditionalurl",/* "official_website",*/ "wikidatacheck").contains(t.title.encoded.toString.toLowerCase))
+      .map(t => new Quad(context.language, hintDataset, subjectUri, templateParameterProperty,
+        t.toWikiText, page.sourceUri, context.ontology.datatypes("xsd:string")))
+
+
+
+    parserFuncQuads ++ templateQuads ++ mappingQuads
+
+  }
+
+  def isNumber(s : String): Boolean = {s.matches("\\d+")}
+
+  def extract_property(str : String, typeOfStr : String) : String = {
+    var answer = ""
+    if ( typeOfStr == "#property") {
+      val pattern = """\{\{#property:([0-9A-Za-z]+)\}\}""".r
+      answer = str match {
+        case pattern(group) => group
+        case _ => ""
+      }
+    } else if (typeOfStr == "#invoke") {
+     val list_words = str.split("""\|""")
+      if (!(list_words(0) == "Wikidata" || list_words(0) == "PropertyLink"))
+        return answer
+      val properties = list_words.filter(s => (s.charAt(0) == 'p' || s.charAt(0) == 'P') && isNumber(s.substring(1)))
+      for ( p <- properties){
+        answer = answer + "/" + p
+      }
+
+      if (answer.size > 0)
+       answer = answer.substring(1)
+
+    }
+    answer
+  }
+  def getPropertyTuples(page : PageNode) : List[(String,String, String)] = {
+    val parserFunctions = ExtractorUtils.collectParserFunctionsFromNode(page)
+
+    val propertyParserFunctions = parserFunctions.filter(p => (p.title.equalsIgnoreCase("#property") && p.children.nonEmpty && !p.children.head.toString.contains("from") && p.parent.isInstanceOf[PropertyNode]))
+
+     (propertyParserFunctions ).map( p =>
+      new Tuple3(p.parent.asInstanceOf[PropertyNode].parent.asInstanceOf[TemplateNode].title.decoded, p.parent.asInstanceOf[PropertyNode].key, extract_property(p.toWikiText, "#property")))
```

Comment:
instead of `extract_property()` `PropertyNode.key` could be used
This function can be useful in cases where the parser fails but we need to figure out these cases first